public/index.html

```diff
@@ -5,6 +5,10 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 
+    <style>
+      @import url('https://fonts.googleapis.com/css?family=Italianno');
```

Comment:
replace with plain old link element.
<link href="https://fonts.googleapis.com/css?family=Italianno:regular" rel="stylesheet" />
also a good idea to add the Roboto font that material-ui depends on in index.html:
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">